server/index.model.js

```diff
@@ -116,8 +112,40 @@ async function insertComment(articleId, newComment) {
     ($1, $2, $3)
     RETURNING *
     `
-
   return db.query(strQuery, insertionParams)
 }
 
-module.exports = { selectTopics, selectArticleById, selectArticles, selectArticleComments, insertComment }
\ No newline at end of file
+async function updateArticle(articleId, changeVotesBy) {
+  if (!changeVotesBy.inc_votes) {
+    return Promise.reject({status: 400, msg: 'bad request'})
+  }
+
+  changeVotesBy = changeVotesBy.inc_votes
+
+  let strQuery = `
+    UPDATE articles
+    SET votes = $1
+    WHERE article_id = $2
+    RETURNING *
+  `
+  const articleTrial = await db.query(`
+  SELECT article_id
+  FROM articles
+  WHERE article_id = $1
+  `, [articleId])
+
+  if (articleTrial.rows.length === 0) {
+    return Promise.reject({ status: 404, msg: 'article not found' })
+  }
+
+  const origVotes = await db.query(`
```

Comment:
Rather than having to do this to update the votes, you can just add the following to strQuery
`SET votes = votes + $1`